game/state/battle/battle.cpp

```diff
@@ -1825,6 +1825,33 @@ void Battle::updateTBEnd(GameState &state)
 			p->updateTB(state);
 		}
 	}
+
+	// Hide units no one in our organisation sees anymore.
+	for (auto it = visibleUnits[currentActiveOrganisation].begin();
+	     it != visibleUnits[currentActiveOrganisation].end();)
+	{
+		bool someoneSees = false;
+
+		for (const auto &u : this->units)
+		{
+			auto &unit = u.second;
```

Comment:
can unit be const?

Reply:
Yes it can.
I'll add const.
Thanks! :)

Reply:
> Yes it can.
> I'll add const.
> Thanks! :)
Привет Кирилл! Спасибо за PR =]
Все разрабы и очень много фанатов сидят в дискорде, если что заходи https://discord.gg/f8Rayre